.github/workflows/docs.yml

```diff
@@ -0,0 +1,49 @@
+name: Docs
+
+on:
+  release:
+    types: 
+      - created
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Checkout docs
+        uses: actions/checkout@v3
+        with:
+          repository: dappnode/DAppNodeDocs
+
+      - name: Set up dappnodesdk
+        run: npm install -g @dappnode/dappnodesdk@${{ github.event.release.tag_name }}
+
+      - name: Download script from sdk
+        run: wget https://raw.githubusercontent.com/dappnode/DAppNodeSDK/master/.github/workflows/scripts/docs.cjs -O docs.cjs
+
+      - name: command_test
+        run: |
+          node docs.cjs
+          rm docs.cjs
+      
+      - name: Commit to docs
+        run: |
+          git add docs/dev/sdk/commands.md
```

Comment:
there are more docs to be tracked of the sdk not only commands:
- Github actions https://docs.dappnode.io/docs/dev/github-actions
- Files references https://docs.dappnode.io/docs/dev/references/docker-compose

We should first ensure that the dappnode sdk contains all this information(commands, github actions and files references), and include this information somehow in the release assets. This is an issue on its own. The release assets could be:
- commands: `.js` files
- files references: `.json` files
- github actions: `.yml` files